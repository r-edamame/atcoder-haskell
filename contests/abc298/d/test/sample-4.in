13
1 9
1 9
1 8
3
2
3
1 4
1 4
2
1 5
1 3
2
3
