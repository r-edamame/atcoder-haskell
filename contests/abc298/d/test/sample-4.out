1998
998
84453
